vehicles/wendover/simple_traffic/simple_wendover_parts.jbeam

```diff
@@ -540,19 +540,19 @@
         {"selfCollision":true},
         {"group":"wendover_trunk"},
         {"nodeWeight":1.5},
-        ["t3r", -0.7641, 1.6268, 0.9438],
-        ["t3", 0.00000, 1.7463, 0.9634],
-        ["t3l", 0.7641, 1.6268, 0.9438],
+        ["t3r", -0.76, 1.6268, 0.9438],
+        ["t3", 0.00000, 1.7463, 0.97],
+        ["t3l", 0.76, 1.6268, 0.9438],
 
-        ["t2r", -0.7579, 2.1222, 0.8803],
-        ["t2", 0.00000, 2.1391, 0.9057],
-        ["t2l", 0.7579, 2.1222, 0.8803],
+        ["t2r", -0.76, 1.99, 0.9],
+        ["t2", 0.00000, 2.0085, 0.9348],
+        ["t2l", 0.76, 1.99, 0.9],
 
-        ["t1r", -0.6206, 2.3488, 0.844],
+        ["t1r", -0.75, 2.3488, 0.844],
         ["t1", 0.00000, 2.3511, 0.8414],
-        ["t1l", 0.6206, 2.3488, 0.844],
+        ["t1l", 0.75, 2.3488, 0.844],
         {"group":""},
-        ["t4", 0.00000, 1.9826, 0.6569],
```

Comment:
Good find, i also wondered where the trunk keeps catching on but didnt figure it out